mongoosepush-rs.yaml

```
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: ReplicaSet
+metadata:
+  name: mongoosepush
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mpush
+  template:
+    metadata:
+      labels:
+        app: mpush
+    spec:
+      containers:
+      - name: mpush
+        image: mongooseim/mongoose-push:kubernetes
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: PUSH_APNS_ENABLED
+          value: "false"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mongoosepush-entrypoint
```

This gets used as a local domain name, so I'd keep it simple, though it's a matter of preference.

```
/ # nslookup mongoosepush-entrypoint
Server:		10.96.0.10
Address:	10.96.0.10:53

Name:	mongoosepush-entrypoint.default.svc.cluster.local
Address: 10.96.5.77

^C
/ # telnet mongoosepush-entrypoint.default.svc.cluster.local 8443
Connected to mongoosepush-entrypoint.default.svc.cluster.local
/ #
```

```suggestion
  name: mongoosepush
```